src/main/plugins/org.dita.pdf2/xsl/fo/commons.xsl

```diff
@@ -459,6 +459,9 @@ See the accompanying license.txt file for applicable licenses.
 
     <xsl:template name="processTopicNotices">
         <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice">
+            <xsl:if test="key('map-id', ancestor-or-self::*[contains(@class, ' topic/topic ')][1]/@id)/ancestor::*[contains(@class,' bookmap/backmatter ')]">
```

Comment:
This works to set it in the attribute set:
```
<xsl:attribute-set name="page-sequence.notice" use-attribute-sets="__force__page__count page-sequence.frontmatter">
    <xsl:attribute name="format">
      <xsl:choose>
        <xsl:when test="key('map-id', ancestor-or-self::*[contains(@class, ' topic/topic ')][1]/@id)/ancestor::*[contains(@class,' bookmap/backmatter ')]">1</xsl:when>
        <xsl:otherwise>i</xsl:otherwise>
      </xsl:choose>
    </xsl:attribute>
  </xsl:attribute-set>
```

But I do wonder if this is going to break some customizations - anybody who overrides `page-sequence.frontmatter` to use format="1" will now get roman numerals for the notices in the front matter, unless / until they override `page-sequence.notice`. 

I don't see a problem with using the `fo:page-sequence` solution in a patch (it's still just a fix), but if you prefer not to, it may be better to go with that solution and do this for 2.4?